src/collection-view.js

```diff
@@ -207,11 +207,13 @@ const CollectionView = Backbone.View.extend({
   // Render children views. Override this method to provide your own implementation of a
   // render function for the collection view.
   render() {
-    if (this._isDestroyed) { return this; }
-    this.triggerMethod('before:render', this);
-    this._renderChildren();
-    this._isRendered = true;
-    this.triggerMethod('render', this);
+    if (!this._isDestroyed) {
```

Comment:
@paulfalgout @scott-w was this the only issue? I can separate this out to let the rest get merged.